src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,94 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    static final int DEFAULT_INITIAL_CAPACITY = 16;
+    static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private Node<K, V>[] map; // наш масив
+    private int size = 0; // кількість елементів
+    private double threshold; // значення критичного заповненення масиву
+
+    public MyHashMap() {
+        map = (Node<K, V>[]) new Node[DEFAULT_INITIAL_CAPACITY];
+        threshold = DEFAULT_INITIAL_CAPACITY * DEFAULT_LOAD_FACTOR;
+    }
+
+    private static class Node<K,V> {
+        V value;
+        MyHashMap.Node<K,V> next;
+        private int hash;
+        private K key;
```

Comment:
why some of the fields are private and other not?

Reply:
done